extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -45,6 +45,9 @@ message PluginConfig {
   // This is deprecated in favor of AccessLogging enum.
   bool disable_server_access_logging = 1 [deprecated = true];
 
+  // Optional. Controls whether to export audit log.
+  bool enable_audit_log = 11;
```

Comment:
Would we have both Full access logs and audit logs at the same time? Because otherwise we can put this in accesslogging enum?

@mandarjog , @bianpengyuan , @douglas-reid

Reply:
This would require four additions to the enum e.g. Audit_Full, Audit_Errors_Only, All_Errors_Only, and All. Or we could also have a separate enum for log type with values Regular, Audit, and All.

Reply:
I think just 2 should be good? 
AUDIT_FULL and AUDIT_ERRORS_ONLY ?

And user will select just one of the 4 options..

Reply:
Talked to David offline.. there is a possibility of having regular and audit logs both based on permissions... so having a bool is better then making this enum complicated I feel..